src/app/content/highlights/components/Highlights.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useSelector } from 'react-redux';
+import myHighlightsEmptyImage from '../../../../assets/MHpage-empty-logged-in.png';
+import htmlMessage from '../../../components/htmlMessage';
+import Loader from '../../../components/Loader';
+import { assertDefined } from '../../../utils';
+import { LinkedArchiveTreeNode } from '../../types';
+import { archiveTreeSectionIsChapter, findArchiveTreeNode } from '../../utils/archiveTreeUtils';
+import { stripIdVersion } from '../../utils/idUtils';
+import { highlightLocationFilters, summaryFilters, summaryHighlights, summaryIsLoading } from '../selectors';
+import { SummaryHighlights } from '../types';
+import * as HStyled from './HighlightStyles';
+import * as Styled from './ShowMyHighlightsStyles';
+
+// tslint:disable-next-line: variable-name
+const NoHighlightsTip = htmlMessage(
+  'i18n:toolbar:highlights:popup:heading:no-highlights-tip',
+  () => <span/>
```

Comment:
i think this is never showing up because you're not forwarding props here

Reply:
Not sure what you want me to do here

Reply:
Ok, I've missed that it is no longer displaying this tip.
I've removed this and just added `FormattedHTMLMessage`